bootstrap.sh

```diff
@@ -6,14 +6,15 @@ if [[ `id -u` -eq 0 ]]; then
 fi
 
 
-declare -a RUBIES=('2.0' '2.1.10' '2.2.5' '2.3' 'jruby-1.7' 'jruby')
-DEFAULT_RUBY="2.0.0-p648"
+IOS_VERSIONS="8.1,8.2,8.3,8.4,9.0,9.1,9.2,9.3,10.0,10.1,10.2,10.3"
+declare -a RUBIES=('2.0' '2.1.10' '2.2.5' '2.3' '2.4' 'jruby-1.7' 'jruby')
+DEFAULT_RUBY="2.4"
 declare -a BREW_PKGS=('git' 'wget' 'mercurial' 'xctool' 'node' \
   'coreutils' 'postgresql' 'postgis' 'sqlite' 'go' 'gpg' 'carthage' \
   'md5deep' 'pyenv' 'tmate' 'cmake' 'swiftlint')
 declare -a PIP_PKGS=('virtualenv' 'numpy' 'scipy' 'tox')
-declare -a NODE_VERSIONS=('6' '5' '4' '0.12' '0.10' '0.8' 'iojs')
-export NVM_VERSION="v0.31.1"
+declare -a NODE_VERSIONS=('8' '7' '6' '5' '4' '0.12' '0.10' '0.8' 'iojs')
```

Comment:
one human's opinion, but I think we can safely trim this down to only _one_ LTS release (`'6'`) since nvm can install anything else on the fly